### DISCUSSION TITLE
predeclared any requires go1.18 or later

### DISCUSSION THREAD

**Discussion by vetjun** [golangci/golangci-lint#4946] [Q&A] [answered]
After using version of v1.60.2 and v1.60.3 lint, lint command is started to give these;
go/pkg/mod/github.com/newrelic/go-agent/v3@v3.30.0/newrelic/sql_driver.go:47:37 -> predeclared any requires go1.18 or later
go/pkg/mod/github.com/newrelic/go-agent/v3@v3.30.0/newrelic/sql_driver.go:51:43: predeclared any requires go1.18 or later

these files are outside of our code base.
last successful lint is being run at version of v1.60.1
it should'nt give error. Or what is changed between v1.60.1 to v1.60.3 to affect that

**Comment by atc0005** [comment]
@vetjun 

You may be encountering the same problem(s) noted here:

- https://github.com/golangci/golangci-lint/issues/4948
- https://github.com/golangci/golangci-lint/pull/4907
- https://github.com/golangci/golangci-lint/issues/4905

**Comment by ldez** [comment, marked as answer]
https://github.com/golangci/golangci-lint/issues/4942

**Comment by vetjun** [comment]
thanks
